app/models/conversion_host.rb

```diff
@@ -38,7 +47,7 @@ def source_transport_method
   def ipaddress(family = 'ipv4')
     return address if address && IPAddr.new(address).send("#{family}?")
```

Comment:
Now `address` is enforced to be `present` and in the format of IP and coming from the `resource.ipaddresses`, do we still need this `ipaddress` method?

Reply:
Probably not, but let's refactor in a separate PR. :)